collective/searchandreplace/browser/pageform.pt

```diff
@@ -40,7 +40,12 @@
             Search and replace
           </h1>
 
-          <div class="documentDescription" tal:content="view/description|nothing">Description</div>
```

Comment:
I would probably keep the [description](https://github.com/collective/collective.searchandreplace/blob/e4a3c6da05e462129fb63196e38a5267b2a0a455/collective/searchandreplace/browser/searchreplaceform.py#L100) for clarity, below your new `h2`.

Reply:
I readded the description below the h2, and added the translation option to be sure